pipeline/alert.go

```diff
@@ -270,6 +283,71 @@ func (a *AlertNode) Flapping(low, high float64) Node {
 	return a
 }
 
+// Send alert to OpsGenie.
+// To use OpsGenie alerting you must first enable the 'Alert Ingestion API'
+// in the 'Integrations' section of OpsGenie.
+// Then place the API key from the URL into the 'opsgenie' section of the Kapacitor configuration.
+//
+// Example:
+//    [opsgenie]
+//      enabled = true
+//      api-key = "xxxxx"
+//      teams = "everyone"
+//
+// With the correct configuration you can now use OpsGenie in TICKscripts.
+//
+// Example:
+//    stream...
+//         .alert()
+//             .opsGenie()
+//
+// Send alerts to OpsGenie using the routing key in the configuration file.
+//
+// Example:
+//    stream...
+//         .alert()
+//             .opsGenie()
+//             .teams('team_rocket')
+//
+// Send alerts to OpsGenie with team set to 'team_rocket'
+//
+// If the 'opsgenie' section in the configuration has the option: global = true
+// then all alerts are sent to OpsGenie without the need to explicitly state it
+// in the TICKscript.
+//
+// Example:
+//    [opsgenie]
+//      enabled = true
+//      api-key = "xxxxx"
+//      teams = "everyone"
+//		recipients = "johndoe"
+//      global = true
+//
+// Example:
+//    stream...
+//         .alert()
+//
+// Send alert to OpsGenie using the default teams, found in the configuration.
+// tick:property
+func (a *AlertNode) OpsGenie() *AlertNode {
+	a.UseOpsGenie = true
+	return a
+}
+
+// The OpsGenie Teams. If not set uses key specified in configuration.
+// tick:property
+func (a *AlertNode) OgTeams(teams string) *AlertNode {
```

Comment:
Then here change the function to:

```
func (a *AlertNode) OgTeams(teams ...string) *AlertNode {
    a.OpsGenieTeams = teams
    return a
}
```

The `...` is Go for multiple args and `teams` will be a list of each of the args.

Then in TICKscript it will look like this:

```
.ogTeams('team1', 'team2', 'team,name,with,commas')
```
 Same for recipients.